logger.Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Build the inference-logger binary
```

Comment:
Should we put our dockerfiles together in CMD or something? 

controller.Dockerfile
admission.Dockerfile
logger.Dockerfile

I've asked knative how they deal with this since they have many.

Reply:
Makes sense but can we punt to a separate PR to keep this focused.

Reply:
They use Ko! Which is a super interesting approach.